src/canmatrix/formats/xls.py

```diff
@@ -114,20 +114,17 @@ def dump(db, file, **options):
                 'Signal Name', 'Signal Function', 'Signal Length [Bit]', 'Signal Default', ' Signal Not Available', 'Byteorder']
     head_tail = ['Value',   'Name / Phys. Range', 'Function / Increment Unit']
 
-    if "additionalAttributes" in options  and len(options["additionalAttributes"]) > 0:
-        additional_signal_colums = options["additionalAttributes"].split(",")
+    if len(options.get("additionalAttributes", "")) > 0:
+        additional_signal_colums = options.get("additionalAttributes").split(",")
     else:
         additional_signal_colums = []#["attributes['DisplayDecimalPlaces']"]
 
-    if "additionalFrameAttributes" in options  and len(options["additionalFrameAttributes"]) > 0:
-        additional_frame_colums = options["additionalFrameAttributes"].split(",")
+    if len(options.get("additionalFrameAttributes","")) > 0:
+        additional_frame_colums = options.get("additionalFrameAttributes").split(",")
     else:
         additional_frame_colums = []#["attributes['DisplayDecimalPlaces']"]
 
-    if 'xlsMotorolaBitFormat' in options:
-        motorolaBitFormat = options["xlsMotorolaBitFormat"]
-    else:
-        motorolaBitFormat = "msbreverse"
+    motorolaBitFormat = options.get("xlsMotorolaBitFormat","msbreverse")
```

Comment:
space after comma. Generally I think that it is good idea to keep the code style rules, because the IDE (pycharm) marks various issues and if we eliminate such superfluous warning sources, we can focus/see real problems. Of course the codestyle check can be disabled, but... yes the python freedom has some costs. But of course it is not my codebase, feel free to ignore me.